src/VendorViews/ViewVendor.js

```diff
@@ -188,6 +188,7 @@ class ViewVendor extends Component {
     const query = location.search ? queryString.parse(location.search) : {};
     const isEdit = query.layer ? query.layer === 'edit' : false;
     const tags = _.get(organization, 'tags.tagList') || [];
+    const categoriesDict = _.get(parentResources, 'vendorCategory.records', []);
```

Comment:
Left it with `parentResources` and `_.get` until `<ViewVendor />` is refactored.

Reply:
literally, it's not a dict, maybe `vendorCategories` is a more appropriate name

Reply:
Updated